k8s/nginx-configmap.yaml

```
@@ -0,0 +1,80 @@
+                return 200 "healthy\n";
+                add_header Content-Type text/plain;
+            }
+            
+            # Serve UI application
+            location / {
+                proxy_pass http://ui_backend;
+                proxy_set_header Host $host;
+                proxy_set_header X-Real-IP $remote_addr;
+                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header X-Forwarded-Proto $scheme;
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)